### PR TITLE
add contributing guideline

### PR DESCRIPTION
**Pull request by KengoTODA**
To guide developers to try to add JUnit test to reproduce. Please review my English is proper to request.

https://github.com/blog/1184-contributing-guidelines

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Contributing guideline
+
+Before you submit issue or pull request, please check following points.
```
Comment (jsotuyod) [inline CONTRIBUTING.md]: > please check the following points.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+## Before reporting problem in detectors
```
Comment (jsotuyod) [inline CONTRIBUTING.md]: > Before reporting a problem with detectors

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+When you found problems like false-positive or false-negative, consider to add a JUnit test case to reproduce.
```
Comment (jsotuyod) [inline CONTRIBUTING.md]: > When you find problems such as false-positives or false-negatives, consider ...

**Comment by jsotuyod** [comment]
Great initiative!

Also of note, we can [add templates for PRs and issues](https://github.com/blog/2111-issue-and-pull-request-templates)

**Comment by KengoTODA** [comment]
@jsotuyod Great, is it better to rename this file as `ISSUE_TEMPLATE.md`?

**Pull request merged**